facade/src/main/java/com/redhat/lightblue/migrator/facade/proxy/FacadeProxyFactory.java

```diff
@@ -100,12 +101,12 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
 
                 switch (t) {
                     case LEGACY: {
-                        log.debug("Not a facade operation, proxy passing to legacy");
+                        log.debug("Method \"{}\" is explicitly annotated to call legacy", method);
```

Comment:
Those are explicitly annotated methods e.g. ```@Direct(target = LEGACY)```. In some cases we may not migrate entire service at once. We can grep for those in code.